qt/scientific_interfaces/Inelastic/Manipulation/InelasticDataManipulationElwinTabModel.cpp

```diff
@@ -117,6 +121,22 @@ std::string InelasticDataManipulationElwinTabModel::createGroupedWorkspaces(Matr
   AnalysisDataService::Instance().remove("specWSnext");
   return workspace->getName() + "_extracted_spectra";
 }
+void InelasticDataManipulationElwinTabModel::setOutputWorkspaceNames(std::string const &workspaceBaseName) {
+  auto elwinSuffix = "_elwin_";
+  m_outputWorkspaceNames["qWorkspace"] = workspaceBaseName + elwinSuffix + "eq";
+  m_outputWorkspaceNames["qSquaredWorkspace"] = workspaceBaseName + elwinSuffix + "eq2";
+  m_outputWorkspaceNames["elfWorkspace"] = workspaceBaseName + elwinSuffix + "elf";
+  m_outputWorkspaceNames["eltWorkspace"] = workspaceBaseName + elwinSuffix + "elt";
+}
+
+std::string InelasticDataManipulationElwinTabModel::getOutputWorkspaceNames() const {
+  std::string outputWorkspaceNames;
+  for (auto const &element : m_outputWorkspaceNames) {
+    outputWorkspaceNames += static_cast<std::string>(element.second) + ',';
+  }
```

Comment:
Would it be possible to use `std::transform` for this? And can we add some tests?